e2e_test/e2e_test.go

```diff
@@ -309,6 +309,7 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 			nfIngressIp                string
 			nfEgressIp                 string
 			externalClientIp           string
+			externalClientDev          string
```

Comment:
Please explain in the commit message shortly why this test was removed in the first place. Well, optimally, you refer to the commit that removed it, and the reason for the removal is there (actually, there isn't a reason given there).

But more importantly. Now that the removal is reverted, please explain why this is undone.

This can be as simple as "This test was known broken and got removed earlier. In the meantime it was fixed, so restore the test". Optimally, explain briefly what the issue was (or which fix was done to make the test work).

But don't write it here. Please write it in the commit message.